packages/berlin/src/data/groups.ts

```diff
@@ -4,23 +4,16 @@ const groups = {
     body: [
       {
         id: 0,
-        text: 'If you are the Lead Researcher or a Solo Researcher, please create a research group.',
-      },
-      {
-        id: 1,
-        text: 'You will get a code that you can share with your collaborators.',
+        text: '(For Lead & Solo Researchers)',
```

Comment:
Should be: "Create a research group and share the access code with your collaborators. Assign proposals to groups from My Proposals."